K8s-deployment/Charts/resource-server/README.md

```diff
@@ -360,6 +360,70 @@ The command removes all the Kubernetes components associated with the chart and
+| `encryption.extraEnvVarsCM`                        | Name of existing ConfigMap containing extra env vars for encryption nodes                            | `rs-env`                                                                                                                                                                                                               |
+| `encryption.extraEnvVarsSecret`                    | Name of existing Secret containing extra env vars for encryption nodes                               | `nil`                                                                                                                                                                                                                  |
+| `encryption.extraVolumes`                          | Optionally specify extra list of additional volumes for the encryption pod(s)                        | `nil`                                                                                                                                                                                                                  |
+| `encryption.extraVolumeMounts`                     | Optionally specify extra list of additional volumeMounts for the encryption container(s)             | `nil`                                                                                                                                                                                                                  |
+| `encryption.sidecars`                              | Add additional sidecar containers to the encryption pod(s)                                           | `{}`                                                                                                                                                                                                                   |
+| `encryption.initContainers`                        | Add additional init containers to the encryption pod(s)                                              | `{}`                                                                                                                                                                                                                   |
+| `encryption.autoscaling.enabled`                   | Enable Horizontal POD autoscaling for Apache                                                         | `true`                                                                                                                                                                                                                 |
```

Comment:
It should be ``Enable Horizontal POD autoscaling for encryption``  and even all the remaining things related to autoscaling params